pkg/controller/account/account_controller.go

```diff
@@ -502,12 +534,20 @@ func (r *ReconcileAccount) assumeRole(
 	roleToAssume string,
 	ccsRoleID string) (awsclient.Client, *sts.AssumeRoleOutput, error) {
 
+	// The role ARN made up of the account number and the role which is the default role name
+	// created in child accounts
+	var roleArn = fmt.Sprintf("arn:aws:iam::%s:role/%s", currentAcctInstance.Spec.AwsAccountID, roleToAssume)
```

Comment:
Similar to `adminAccessArn` const, we could move this string into its own const like `roleArnFormat`

Reply:
Hmm, I'm not sure if I fully understand what you're trying to suggest.  Do you mean make a separate var like this: `roleArnFormat := "arn:aws:iam:%s:role/%s"`?

I think I'd have to see it in a separate PR in order to fully understand the scope of the change, but also I'm not sure if it's worth pulling out, if that will make the code easier to read or harder.  🤔   If you want, we can create a card for this and I think this would be a good chance for you to get familiar with the operator code as a first small PR?

Reply:
Yup no worries, given this has been merged now anyways :slightly_smiling_face: 

The change was very minor/non-invasive, just to move the string into a const similar to [adminAccessArn](https://github.com/openshift/aws-account-operator/blob/master/pkg/controller/account/account_controller.go#L74). I can agree that it might make the code harder to read having to jump up to the top of the file - it was mainly from a code cleanliness viewpoint not to have hardcoded strings set in functions. I don't think it merits a card tbh - was only a small comment :smile: